docs/examples/files.md

```diff
@@ -190,11 +191,10 @@ class Person(BaseModel):
     email: EmailStr
 
 
-with open('people.csv') as f:
-    reader = csv.DictReader(f)
-    people = [Person.model_validate(row) for row in reader]
-
-print(people)
+csv_lines = pathlib.Path('people.csv').read_text().splitlines()
+reader = csv.DictReader(csv_lines)
+people = [Person.model_validate(row) for row in reader]
```

Comment:
I think I prefer the existing form, saves some function calls and is more readable.